robot/Cumulus/tests/browser/affiliations/change_secondary_as_primary_affiliation.robot

```diff
@@ -1,29 +1,39 @@
 *** Settings ***
 
 Resource        robot/Cumulus/resources/NPSP.robot
-Suite Setup     Open Test Browser
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/ContactPageObject.py
+...             robot/Cumulus/resources/AccountPageObject.py
+...             robot/Cumulus/resources/AffiliationPageObject.py
+...             robot/Cumulus/resources/NPSP.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
 Suite Teardown  Delete Records and Close Browser
 
+***Keywords***
+Setup Test Data
```

Comment:
This isn't a comment, but a question. I've seen code like this in several tests that I've reviewed lately. I'd like to find a way to make this less tedious to write.  It's cumbersome to call a keyword, save the value to a variable, then call `set suite variable` on that variable, and do that several times before calling `Store Session Record`.

It would be nice if we could do that all on one line. If you have a chance, I'd love to hear your opinions on how we could make creating test data easier.  

For example, maybe we could write our own keyword that works like `Set suite variable` but which takes a keyword instead of regular data, so you could do something like this: 

```
Create suite data  &{account}  API Create Organization Account
Create suite data  &{contact}  API Create Contact  email=automation@example.com
```

We could even make it so that you can create multiple variables at once, sort of like how `Run keywords` works

```
Create suite data
...       &{account}  API Create Organization Account
...  AND  &{contact}  API Create Contact  email=automation@example.com
```
I'm just brainstorming here -- if you have ideas for how would could make the creation of test data easier, I'd like to hear them.